gradle.properties

```diff
@@ -24,6 +24,7 @@ android.nonTransitiveRClass=true
 BASE_URL="https://kizzy-api.cjjdxhdjd.workers.dev"
 org.gradle.unsafe.configuration-cache=true
 android.defaults.buildfeatures.buildconfig=true
-android.nonFinalResIds=false
+android.nonFinalResIds=true
```

Comment:
Is this necessary?

Reply:
Not really, it just really helps with incremental compilation time. [It is the default Gradle config](https://developer.android.com/build/releases/past-releases/agp-8-0-0-release-notes) and **only** breaks stuff [if you use Resource IDs in switch cases](https://stackoverflow.com/questions/69120822/resource-ids-will-be-non-final-by-default-in-android-gradle-plugin-version-8-0)

Reply:
This shouldn't be in this pr then i guess